backend/organization/utility/notification.py

```diff
@@ -43,4 +43,41 @@ def create_project_follower_notification(project_follower):
         if not member['user'] == project_follower.user.id:
             user = User.objects.filter(id=member['user'])[0]
             create_user_notification(user, notification)
-            send_project_follower_email(user, project_follower, notification)
\ No newline at end of file
+            send_project_follower_email(user, project_follower, notification)
+
+
+def create_project_join_request_notification(requester,project_admins,project):
+    """
+    Creates a notification about a joining request from a requester to a project admin. 
+    :param requester: UserProfile object of the user who's sent the request 
+    :type requester: UserProfile
+    :param project_admin: Iterable UserProfile object of the project administrators
+    :type project_admin: List(UserProfile)
+    
+    """
+    requester_name = UserProfile.objects.filter(user=requester).first().name
+    notification = Notification.objects.create(
+        notification_type=9,
```

Comment:
Rather than number I would prefer using `Notification.JOIN_PROJECT_REQUEST` here. Much better and cleaner to understand.

Reply:
This is also fine for now. No need to make code changes